developer-docs-site/docs/guides/move-guides/book/standard-library.md

```diff
@@ -1,542 +1,3 @@
-# Standard Library
+# Libraries and Aptos Framework
 
-The Move standard library exposes interfaces that implement the following functionality:
-* [Basic operations on vectors](#vector).
-* [Option types and operations on`Option` types](#option).
-* [A common error encoding code interface for abort codes](#errors).
-* [32-bit precision fixed-point numbers](#fixed_point32).
-
-## vector
```

Comment:
I think we can keep most of the vector?

Also cc @lightmark for adding the new data structures here in the near future